app/vmselect/promql/eval.go

```diff
@@ -574,7 +576,9 @@ func evalRollupFuncWithMetricExpr(ec *EvalConfig, name string, rf rollupFunc, me
 	sq := &storage.SearchQuery{
 		MinTimestamp: start - window - maxSilenceInterval,
 		MaxTimestamp: ec.End + ec.Step,
-		TagFilterss:  [][]storage.TagFilter{me.TagFilters},
+		TagFilterss: [][]storage.TagFilter{
+			*(*[]storage.TagFilter)(unsafe.Pointer(&me.TagFilters)),
```

Comment:
Is this OK?

It's the same memory layout, so I think it should be fine, and I wanted to avoid the overhead of copying